modules/vsphere-problem-detector-viewing-events.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+[id="vsphere-problem-detector-operator-view-events_{context}"]
+= Viewing the events from the vSphere problem detector
+
+After the vSphere Problem Detector Operator runs and performs the configuration checks, it creates events that can be viewed from the command line or from the monitoring user interface.
+
+.Procedure
+
+* To view the events by using the command line, run the following command.
++
+[source,terminal]
+----
+$ oc get event -n openshift-cluster-storage-operator \
+   --sort-by={.metadata.creationTimestamp}
+----
++
+.Example output
+----
+16m     Normal    Started             pod/cluster-storage-operator-xxxxx         Started container vsphere-problem-detector
+16m     Normal    Created             pod/cluster-storage-operator-xxxxx         Created container vsphere-problem-detector
+16m     Normal    LeaderElection      configmap/vsphere-problem-detector-lock    cluster-storage-operator-xxxxx became leader
+----
+
```

Comment:
You have mentioned the monitoring user interface in the intro to this module but no steps are included as to how. I think it would be useful to have an additional bullet point stating how. Please replace `<etc>` with the appropriate page in the web UI:

` * To view the events by using the {product-title} web console, naviage to *Monitoring* -> <etc>`